src/language/concatAST.ts

```diff
@@ -0,0 +1,14 @@
+import { ASTNodeKind, DocumentNode, TypeDefinitionNode } from '../language/ast';
+
+/**
+ * Provided a collection of ASTs, presumably each from different files,
+ * concatenate the ASTs together into batched AST, useful for validating many
+ * GraphQL source files which together represent one conceptual application.
```

Comment:
I think we should get rid of GraphQL mentions in our code 😅

Reply:
Hm